scripts/mood.coffee

```diff
@@ -0,0 +1,19 @@
+# Description:
+#   Hubot will flip your fucking mood.
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   mood - declare your daily mood
+#
+# Author:
+#   max-sec
+
+module.exports = (robot) ->
+  robot.respond /(mood)/i, (msg) ->
+    result = msg.random ["shorturl.at/ceiVZ", "shorturl.at/hiQRT"]
```

Comment:
if they're urls, they should start with the protocol - and also why a shortner, wouldn't that just make it even more likely to eventually fail?

Reply:
Temp use for the lols. I'll host them myself and update later